src/main/scala/com/danielasfregola/twitter4s/entities/DirectMessageEventList.scala

```diff
@@ -0,0 +1,7 @@
+package com.danielasfregola.twitter4s.entities
+import com.danielasfregola.twitter4s.entities.streaming.UserStreamingMessage
+
+final case class DirectMessageEventList(events: List[DirectMessageEvent],
+                                        apps: Option[Map[String, Apps]],
```

Comment:
Why an `Option[Map[.., ...]]` and not just a `Map[..., ...]`?

Reply:
done